### ISSUE TITLE
1.13.2.1 broken

### ISSUE THREAD

**Issue by fgsfdss** [Nevcairiel/Inventorian#3]
Update broke the addon, getting blank bags and bank now. Reverting to 1.13.2.0 fixed it.

**Comment by Nevcairiel** [comment]
1.13.2.1 works just fine here. I tested various classes to be sure without any issues.

Unless of couse you are using  1.13.2.x versions on Retail, that would not work.

**Comment by fgsfdss** [comment]
I'm on the stress test. New level 1 character, only addon enabled.

**Comment by Nevcairiel** [comment]
I'm running that version just this second, on either chars i've had for a few days or even a brand new one, no issues whatsoever.

I can only guess that perhaps your download corrupted or something like that. I tried both the versions published here on GitHub or on WoWAce and it runs fine.

**Comment by fgsfdss** [comment]
I got the latest version from Curseforge. Results here:
![image](https://user-images.githubusercontent.com/5033279/62866200-86c9a880-bd10-11e9-8376-3b098a0cb1c0.png)
I tried downloading from Github as well, that one didn't really work at all. Default bags showed up. Of course I changed the folder name and made sure it was enabled in game.
Reverted to 1.13.2.0:
![image](https://user-images.githubusercontent.com/5033279/62866324-cc867100-bd10-11e9-8df2-c66615b95fe9.png)

**Comment by Nevcairiel** [comment]
If you have the brush icon in the top, that means you got the Retail version of the addon. Otherwise, it cannot exist, since I deleted it.

Not sure how that happened, but it did. Are you using Twitch client?
When downloading mods for classic you need to be careful to get the classic version, because by default it'll always try to give you the retail version.

**Comment by fgsfdss** [comment]
No, I did a direct download. Made sure to pick the 1.13.2.1 as well. Although it may be possible that the top right download button links to a different version than the central one. I used the top right. That one goes to /download, whereas the central one goes to /download/2759235.
![image](https://user-images.githubusercontent.com/5033279/62877628-c0f27480-bd27-11e9-9240-db7c02835951.png)
Unfortunately I can't check anymore.
